.github/actions/examples/action.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: Run examples
+description: Composite workflow for running examples
+runs:
+  using: composite
+  steps:
```

Comment:
This is the weird thing of calling composite actions _within_ the same repo that they are constructed. You require the `checkout` step, which you don't if you call them external to the repo.

Not a big problem, just annoying, and unintuitive. I think in a reusable workflow, we wouldn't have that issue.

Reply:
TIL: reusable workflows

I had never tried reusable workflows before, but I agree, it is much more intuitive, so I have switched things up.

Reply:
I don't know why the two have such different names.

I have all my composite actions defined here https://github.com/paddyroddy/.github/tree/main/actions, but within a repo I do think reusable probably makes more sense